docs/built-in-models/encrypted_dataframe.md

```diff
@@ -0,0 +1,111 @@
+# Working with Encrypted DataFrames
+
+Concrete ML builds upon the pandas data-frame functionality by introducing the capability to construct and perform operations on encrypted data-frames using FHE. This API ensures data scientists can leverage well-known pandas-like operations while maintaining privacy throughout the whole process.
+
+Potential applications include:
+
+- Encrypted storage of tabular datasets
+- Joint data analysis efforts between multiple parties
+- Data preparation steps before machine learning tasks, such as inference or training
+- Secure outsourcing of data analysis to untrusted third parties
+
+## Encrypt and Decrypt a DataFrame
+
+To encrypt a pandas `DataFrame`, use the `encrypt_from_pandas` function. First, a client engine, which manages keys, must be constructed using `load_client`.
```

Comment:
this is wrong now: we need to create a `ClientEgine` instance and use its `encrypt_from_pandas` method to crate a `EncryptedDataFrame` object